crates/storage/provider/src/providers/database/provider.rs

```diff
@@ -2148,7 +2148,17 @@ impl<TX: DbTxMut + DbTx> BlockWriter for DatabaseProvider<TX> {
 
             let start = Instant::now();
             self.tx.put::<tables::Transactions>(next_tx_num, transaction.into())?;
-            transactions_elapsed += start.elapsed();
+            let elapsed = start.elapsed();
+            if elapsed > Duration::from_millis(100) {
```

Comment:
this might be too short and could potentially spam output?

Reply:
agree, increased to 1s